main.c

```diff
@@ -0,0 +1,73 @@
+// TODO(wendi): init external oscillator, then change this to 12MHz
+#include <stdint.h>
+#include "mcc_generated_files/mcc.h"
+#include "canlib/pic18f26k83/pic18f26k83_timer.h"
+#include "process_fill.h"
+#include "leds.h"
+
+// variable that stores current fill level
+volatile uint8_t fill_level = 0;
+
+void pin_init() {
+    // set RA as an output port (for LEDs)
+    TRISA = 0b00000000;
+
+    // set RB and RC as input ports
+    TRISB = 0b11111111;
+    TRISC = 0b11111111;
+
+    // enable interrupt-on-change for RB and RC on falling edge
+    IOCBN = 0b11111111;
```

Comment:
It seems a little interesting that you are enabling the interupt on change for RB6 and RB7, which aren't used.